VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 |---------|-------------|-----------------|----------------------|
+| 5.0.0-beta.6 | 4.10.1 | 5.4.1 | 4.1.2 |
```

Comment:
Formatting here is off, but ultimately what matters is how it's rendered I guess

Reply:
Yeah, it was difficult to automate this to match what we were doing before, since the number of spaces depended on the version number... In the end, I thought we would just care about how it looks when rendered. We can revisit this if necessary in the future.

Reply:
Agreed!